packages/opds-web-client/src/components/__tests__/UrlForm-test.tsx

```diff
@@ -33,9 +33,9 @@ describe("UrlForm", () => {
     );
 
     let form = wrapper.find("form");
-    let input = wrapper.find("input").get(0) as any;
-
-    input.value = "some url";
+    
+    // set the value for the input
+    wrapper.setProps({ collectionUrl: "some url" });
```

Comment:
It's good to test using the props, but it would still be good to also check that something typed in works.

Reply:
Yes, I figured out how to do so later on and forgot to update this one.